### DISCUSSION TITLE
how to add to organization billing users

### DISCUSSION THREAD

**Discussion by Jaroslaveta** [opengisch/QField#3921] [Q&A]
Hi, now when selecting up Subscriptions select change 
we see only Included members 2 can we change it to 5 members 
because in the organization now are 5 members

**Comment by ZsanettMed** [comment]
Hi Jaroslaveta, 
First make an account for each member, then add them as a member to the organization. An organization includes at least two users, this amount of users will be billed each month, no matter if there were active. Further members will be billed only if they actively use QFieldCloud in a given month.
I hope I could help!
Best, 
Zsanett
